<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конфигуратор компьютера</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 189, 190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="component_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Компоненты</string>
    </property>
   </widget>
   <widget class="QLabel" name="history_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>История</string>
    </property>
   </widget>
   <widget class="QLabel" name="pc_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сборка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pc_saveBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>291</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="component_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="component_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>431</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="component_addBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QListWidget" name="pc_components">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>621</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pc_sum">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pc_name">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="history_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="history_loadBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>171</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="history_deleteBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>171</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="component_deleteBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 1px solid rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>МАГАЗИН &quot;ОНЛАЙН КОМПЬЮТЕР&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="exitBtn"/>
    <addaction name="user"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="exitBtn">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="user">
   <property name="text">
    <string>User</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
